### ISSUE TITLE
observable properties for methods?

### ISSUE THREAD

**Issue by meganrwong** [ANZSoilData/def-au-scma#8]
It may be useful that if observable properties the procedures measured were in this vocabulary that they could be encoded. 

Reviewed if could be easily pulled out

It seems not: 
- The chapter header tells you the high level obsv prop or substance/species being measured - but not always 

- Each chapter has a section before the coded procedures.
This section sometimes talks about the property being observed at the start, and sometimes not. 
Sometimes, a description of the observable property being measured can be scattered in different spots through the section
Sometimes, the intro talks about an observable property that is not being measured, before going into what is being measured
In some cases the section has sub-headers that are the observable property, but there are no definitions with this

- Each individual procedure talks about the property being observed but not always explicitly or in a consistent way between procedures

In conclusion I think the obsv properties each procedure measures can not be pulled out easily. 

If there is value in linking the procedures in this collection to the obsv properties they measure, then this would require building on this vocabulary through linking to external obsv property vocabularies and would require expert input/governance. It is outside of first task of representing green book as is
